specification.rst

```diff
@@ -1543,6 +1546,10 @@ control the testing logic, with allowed values:
 -  "none" - element only tests "true" when none of the conditions test "true"
    for any given test value
 
+The output of ``cs:choose`` (i.e. the output of the matching ``cs:if``, 
+``cs:else-if``, or ``cs:else``) *does* have delimiters from a parent delimiting 
```

Comment:
ditto

Reply:
In each case ancestor would be technically correct if you did `group with delimiter > choose > if > choose > if` hmm

Reply:
Nearest ancestor delimiting element does it I think

Reply:
Because I defined delimiting element to be even if there is no delimiter set on it, yeah